src/util/uint.rs

```diff
@@ -393,7 +393,7 @@ macro_rules! construct_uint {
             }
         }
 
-        impl core::fmt::Debug for $name {
+        impl core::fmt::Display for $name {
```

Comment:
This one will conflict with #1197 where I mentioned `Display` should not be implemented for `UintX` but `LowerHex` instead.